nodejs/image.bzl

```diff
@@ -129,21 +122,30 @@ def nodejs_image(
         **kwargs
     )
 
+    layers = [
```

Comment:
any reason this now is after the nodejs_binary call above (if so, could you briefly document)